Gemfile

```diff
@@ -17,6 +17,7 @@ gem "github-pages", group: :jekyll_plugins
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
+  gem "rouge"
```

Comment:
Hadrien wrote a few equations for the articles that deal with indicators. For the time being, the equations are written in plain text, but we assumed that using [TeX](https://en.wikipedia.org/wiki/TeX) will improve the readability of a few equations, especially the ones that contains fractions, for example : `LTV / CAC`.

Since [Rouge seems to support TeX](https://github.com/rouge-ruby/rouge/blob/master/docs/Languages.md), I added the Rouge plugin to the Gemfile in order to get ready for converting our plain text equations to TeX ones—which I haven't started yet. This might lead nowhere and we might decide to stick with plain text later down the road, but at least we can try a few things now.

Reply:
I think `rouge` will *only* perform syntax coloring.

If you want a TeX equation to look good, what you want is probably [MathJax](https://www.mathjax.org).

Back in the days, it was not so trivial to setup so ping me if you need help at some point.